helm/examples/azure/hpcc-azurefile/values.yaml

```diff
@@ -11,31 +12,36 @@ planes:
   category: dali
   #sku: "Standard_LRS"
   #shareName: dalishare
+  #volumeId: "100-dali"
```

Comment:
not sure of the significance or meaning to the end-user of the "100-" prefixes?

Reply:
I got error if without any digits. But Kubernetes CSI driver docs just says it should be unique in the cluster (Actually not sure the "cluster" means AKS or user app cluster).